scripts/release/preparepackages.js

```diff
@@ -56,7 +56,20 @@ const tasks = new Listr( [
 
 			return Promise.reject( 'Aborted due to errors.\n' + errors.map( message => `* ${ message }` ).join( '\n' ) );
 		},
-		skip: cliArguments.nightly
+		skip: () => {
+			// Nightly releases are not described in the changelog.
+			if ( cliArguments.nightly ) {
+				return true;
+			}
+
+			// When compiling the packages only, do not validate the release.
+			if ( cliArguments.compileOnly ) {
+				return true;
+			}
+
+			return false;
+		},
+		retry: 0
```

Comment:
Why is `retry: 0` change added into several places?

Reply:
To revert.